3.3/.sti/bin/assemble

```diff
@@ -5,12 +5,42 @@ source .bashrc
 
 set -e
 
-echo "---> Installing application source"
+echo "---> Copying application source ..."
 cp -Rf /tmp/src/* ./
 
-echo "---> Building your Python application from source"
-if [ -f setup.py ]; then
-    python setup.py install --user
-elif [ -f requirements.txt ]; then
-    pip install --user -r requirements.txt
+# If no requirements given, assume project root is installable
+# i.e., contains a valid `setup.py`.
```

Comment:
Are we aiming to support repositories that neither have a `requirements.txt` nor `setup.py`?
I added a test case for trying to build a repo without either `setup.py` or `requirements.txt` (build should fail).

Those files are two good indicators that the repo is "Python-compatible", and we could use that when auto-detecting, for instance, in `osc new-app`.

In current master tip there is no else clause to that if statement. When neither `setup.py` nor `requirements.txt` is present, the `assemble` script will only copy the source code to a new location. And `run` will have only a single option to do it's job: `exec python -u ${APP_FILE}`. How realistic is that that it will launch an HTTP server with a working application?